helper/general_objects/schemas.go

```diff
@@ -346,6 +346,10 @@ func DefinitionAttributeSchema(excludeTypes []string, excludeFields []string) ma
 		},
 	}
 
+	if forceNew {
+		schema["type"].ForceNew = true
```

Comment:
what is this?

Reply:
`ForceNew` set to true indicates that when changing the value of this field, terraform will have to destroy and recreate the resource again ([see](https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#forcenew))
But you also cannot set it to `false` so I do it this way